### ISSUE TITLE
Search @

### ISSUE THREAD

**Issue by Charlie1103** [sabrogden/Ditto#89]
In the previous version of Ditto, I was able to search for copied text that started with @.  Now it doesn't seem to work anymore.  Is there a setting or something I can change to be able to search with @ ?

Thanks.
-Charlie

**Comment by sabrogden** [comment]
What search option do you have selected?  ... - search options - contains/regular exp/wild card?  I'm only seeing this not work with regular exp because it has a meaning in regular expression.  Make sure you have contains search selected.

**Comment by Charlie1103** [comment]
I have "Search Description" and "Contains Text Search" selected. 
![image](https://user-images.githubusercontent.com/80854535/111883260-2c7ed800-8988-11eb-9c05-34b4dbfb3273.png)
I have noticed something else that may be the issue.  I can't search for anything when I have the Shift key down (so it is not just the @ sign).  BUT, if I click down in the Search field at the bottom, then I can use the Shift key, and find the @, etc.  One other point is that I have set my Activate Ditto shortcut to Ctrl-Shift-V.  (Not sure if that has anything to do with it.)
![image](https://user-images.githubusercontent.com/80854535/111883413-1f161d80-8989-11eb-8f73-a44cd0dd325b.png)

Thanks.
-Charlie

**Comment by scottbrogden-iheartmedia** [comment]
If you put focus to the search box and you type @ does it work?  I'm seeing it not put focus to the search box when focus is on the main list and I type @.  Is that what you are talking about.

**Comment by sabrogden** [comment]
If the problem was that Ditto wasn't setting focus to the search text box when typing @ that is fixed in the latest, https://ditto-cp.sourceforge.io/beta/

**Comment by Charlie1103** [comment]
Yes, the problem is that the focus does not go to the Search box.  I downloaded and installed the Beta.  It works!  Thanks!

-Charlie

**Comment by ttk** [comment]
Just ran into this issue.  Seems like the problem happens when you hold down the SHIFT key and type to search.  

Referencing the fix here: https://github.com/sabrogden/Ditto/commit/faf850d99c6990415ec885eb0fd433156efb7cb0

**Comment by nematodos** [comment]
> I have "Search Description" and "Contains Text Search" selected. ![image](https://user-images.githubusercontent.com/80854535/111883260-2c7ed800-8988-11eb-9c05-34b4dbfb3273.png) I have noticed something else that may be the issue. I can't search for anything when I have the Shift key down (so it is not just the @ sign). BUT, if I click down in the Search field at the bottom, then I can use the Shift key, and find the @, etc. One other point is that I have set my Activate Ditto shortcut to Ctrl-Shift-V. (Not sure if that has anything to do with it.) ![image](https://user-images.githubusercontent.com/80854535/111883413-1f161d80-8989-11eb-8f73-a44cd0dd325b.png)
> 
> Thanks. -Charlie

Hola, disculpe, ¿me puede decir cómo usar la función Wildcard Search?

**Comment by nematodos** [comment]
> I have "Search Description" and "Contains Text Search" selected. ![image](https://user-images.githubusercontent.com/80854535/111883260-2c7ed800-8988-11eb-9c05-34b4dbfb3273.png) I have noticed something else that may be the issue. I can't search for anything when I have the Shift key down (so it is not just the @ sign). BUT, if I click down in the Search field at the bottom, then I can use the Shift key, and find the @, etc. One other point is that I have set my Activate Ditto shortcut to Ctrl-Shift-V. (Not sure if that has anything to do with it.) ![image](https://user-images.githubusercontent.com/80854535/111883413-1f161d80-8989-11eb-8f73-a44cd0dd325b.png)
> 
> Thanks. -Charlie

Hi, excuse me, can you tell me how to use the Wildcard Search function?

**Comment by Chealer** [comment]
@Charlie1103: If this is solved, please mark this ticket as such.
